Fifth-Order-Ignorant/Baobab
# [FIF-66] Rename message to post

## [FIF-66] Rename message to post

Fixes: #


### Changes
Rename messages to posts on both the front and backend.

### Testing

**Manually QA'd by doing the following:**

1. Testing to see the application still works on all pages (mainly ``/profile/`` and ``/feed/``):
```
npm run build --workspace=common
npm run start --workspace=server
npm run dev --workspace=client
```
2. Ran linter to see if there are errors.
```
npm run lint --if-present --workspaces
```
3. Running the generated documentation for inconsistencies (see ``localhost:6006`` and ``localhost:3001/docs``:
```
npm run storybook --workspace=client
```

**Automated QA**: Run ``npm run test --workspace=server``

**Note: Merge this PR once the automated testcase pull has been applied.**
